napari/_qt/qt_plugin_list.py

```diff
@@ -0,0 +1,199 @@
+"""Provides a QtPluginSorter that allows the user to change plugin call order.
+"""
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtWidgets import (
+    QComboBox,
+    QDialog,
+    QListWidget,
+    QListWidgetItem,
+    QLabel,
+    QSizePolicy,
+    QVBoxLayout,
+    QWidget,
+)
+from ..plugins import plugin_manager as napari_plugin_manager
+from ..plugins.utils import permute_hook_implementations, HookOrderType
+from pluggy.manager import PluginManager, _HookCaller
+from typing import Optional
+
+
+class QtHookImplItem(QListWidgetItem):
+    """Item in a QtHookImplListWidget."""
+
+    def __init__(self, hookimpl, parent=None):
+        self.plugin_name = hookimpl.plugin_name
+        if self.plugin_name == 'builtins':
+            self.plugin_name = 'napari built-in'
+        super().__init__(self.plugin_name, parent)
+        self.hookimpl = hookimpl
+        self.setBackground(Qt.black)
```

Comment:
Here also, the theming within the class seems to go against our existing design principles.